### PR TITLE
Resolve ansible-lint issues and other cleanup changes

### PR DESCRIPTION
**Pull request by pmackay**
@mkllnk would appreciate it if you could review this.

### CODE REVIEW

File: roles/deploy/handlers/main.yml [hunk 1]
```diff
@@ -42,3 +44,10 @@
     state: started
   become: yes
   become_user: "{{ unicorn_user }}"
+
+- name: restart nginx
```
Comment (mkllnk) [inline roles/deploy/handlers/main.yml]: Is this re-starting nginx or only starting it when it's not running yet?

Comment (pmackay) [inline roles/deploy/handlers/main.yml]: Well ansible just uses state: started so could it be either? Would you prefer if these tasks were called start xxx?

**Comment by pmackay** [comment]
>Should we mention how to use ansible-lint in the README? Can we add that to Travis?

I've added some notes on it, please check. 

This is already run in Travis.

File: README.md [hunk 2]
```diff
@@ -29,6 +29,20 @@ You will need Ansible on your machine to run the playbooks.
...
+## Code quality
+
+Install [ansible-lint](https://github.com/willthames/ansible-lint) by running:
+```
+pip install ansible-lint
+```
+
+Run the checks using:
+```
+ansible-lint site.yml
```
Comment (mkllnk) [inline README.md]: I think `--exclude=community` is missing. Otherwise you get a lot of errors.

Comment (mkllnk) [inline README.md]: Added it myself. :-)

**Pull request merged**